src/components/common/Tabs/EditProfileTab/styles.scss

```diff
@@ -0,0 +1,13 @@
+.edit-profile-label {
+  font-size: 11px;
+  line-height: 13px;
+  text-align: center;
+  letter-spacing: 1.65px;
+}
+
```

Comment:
This two styles are very similar. In my project I've set a default font style that includes line-height, letter-spacing and font-family. I apply this style to the body in order to have a default font style and not having to repeat it in every component or element. 0.1px of difference between your edit-profile-label class and your input-placeholder class is meaningless, don't you think?

Reply:
It's pretty similar but I found that changing it messes my whole app :s